
<launch>

    <node name="slamdunk_fusion" output="screen" pkg="duo_slam" type="slamdunk_fusion">

        <param name="hz" value="30"/>
        <param name="cores" value="3"/>
        <param name="viz" value="false"/>
        <param name="build_environment" value="true"/>
        <param name="tracker_start_time" value="4"/>

        <param name="camera_rgb_topic" value="/camera/rgb/image_raw"/>
        <param name="camera_depth_topic" value="/camera/depth/image_raw"/>
        <param name="camera_pose_topic" value="/camera/pose"/><!-- SLAM OUTPUT TOPIC -->

        <!--        <param name="fx" value="542.461710"/>
               <param name="fy" value="543.536535"/>
               <param name="cx" value="311.081384"/>
               <param name="cy" value="236.535761"/>-->
        
        <param name="fx" value="521.265683"/>
        <param name="fy" value="519.874996"/>
        <param name="cx" value="317.715937"/>
        <param name="cy" value="239.811139"/>
        
        <param name="rows" value="480"/>
        <param name="cols" value="640"/>
        <param name="depth_scale_factor" value="0.0002"/>


        <param name="rings" value="3"/>
        <param name="kf_overlapping" value="0.8"/>
        <param name="loop_inference" value="false"/>
        <param name="doicp" value="false"/>
        <param name="icp_distance" value="0.2"/>
        <param name="icp_normal" value="30.0"/>
        <param name="verbose" value="false"/>

        <param name="features" value="surfext"/>
        <param name="winl" value="6.0"/>
        <param name="feat_redux" value="false"/>

        <param name="tsdf_size" value="2.5"/>
        <param name="tsdf_resolution" value="200"/>
        <param name="tsdf_trunc_dist_pos" value="0.03"/>
        <param name="tsdf_trunc_dist_neg" value="0.03"/>
        <param name="raytrace_reduction" value="1"/>
        
        
    </node>


</launch>
